### DISCUSSION TITLE
PostgreSQL如何导出数据

### DISCUSSION THREAD

**Discussion by myrzx** [swuecho/chat#368] [Q&A] [answered]
我想用用户对话记录来微调本地模型（已告知会使用数据），看了下railway里面是用PostgreSQL作为数据库，但是没找到下载数据库文件的方法。

**Comment by swuecho** [comment, marked as answer]
railway 你找到 database URL 然后pg dump 就可以了 具体搜一下

**Reply by myrzx** [reply]
好的，非常感谢！
